packages/agents/src/agents/Evo/Evo.spec.ts

```diff
@@ -112,32 +110,43 @@ describe('Evo Test Suite', () => {
     }
   }
 
+  test.only("evo selects correct agent", async () => {
+    const goal =
+      "How much was spent on utilities in total ? Write the answer in an output.txt file.";
+    const { llm, chat } = createEvo("selects-agent-correctly");
+    const evoConfig = EVO_AGENT_CONFIG([
+      DEVELOPER_AGENT_CONFIG,
+      RESEARCHER_AGENT_CONFIG,
+      DATA_ANALYST_AGENT,
+    ]);
+    const messages = evoConfig.initialMessages({ goal });
+    messages.forEach((message) => {
+      chat.persistent(message.role, message.content);
+    });
+
+    const functionDefinitions = evoConfig.functions.slice(2).map((fn) => fn.getDefinition());
+    for (let i = 0; i<20; i++) {
+      const response = await llm.getResponse(
+        chat.chatLogs,
+        functionDefinitions,
+        { temperature: 0.5 }
```

Comment:
Unless we're using this in Evo, I don't think we should be using it here cause it's not a good representation of what happens when you use evo

Reply:
okay just removed it